### PR TITLE
Ensure ntpd is running

### PR DESCRIPTION
**Pull request by jgrund**
with the advent of `chronyd`, `ntpd`, which IML currently relies on may
be inactive.

Make sure we stop and disable chronyd on the manager.

Signed-off-by: Joe Grund <jgrund@whamcloud.io>

### CODE REVIEW

File: chroma_core/lib/service_config.py [hunk 1]
```diff
@@ -467,7 +471,7 @@ def _check_db_space(self, required_space_gigabytes):
             return error_msg
 
     def _restart_pgsql(self):
-        postgresql_service = ServiceControl.create("postgresql")
+        postgresql_service = ServiceControlEL7.create("postgresql")
```
Comment (jgrund) [inline chroma_core/lib/service_config.py]:
```suggestion
        postgresql_service = ServiceControl.create("postgresql")
```

Comment (johnsonw) [reply chroma_core/lib/service_config.py]: Curious as to why this was reverted?

Comment (jgrund) [reply chroma_core/lib/service_config.py]: It was causing an error in EFS.

Comment (johnsonw) [reply chroma_core/lib/service_config.py]: 👍

**Pull request merged**